apps/condo/pages/onboarding.tsx

```diff
@@ -118,6 +126,11 @@ const OnBoardingPage: IOnBoardingIndexPage = () => {
                     <ServiceSubscriptionWelcomePopup/>
                 )
             }
+            {
+                !isLoading && organizationImportRemoteSystem === SBBOL_IMPORT_NAME && !isAnySbbolOnBoardingStepCompleted && (
```

Comment:
It looks more closer to human language with `some` utility from Lodash:

```suggestion
                !isLoading && organizationImportRemoteSystem === SBBOL_IMPORT_NAME && !some(otherSteps, 'completed') && (
```